libfuse/dir.go

```diff
@@ -134,6 +134,26 @@ func (f *Folder) forgetNode(node libkbfs.Node) {
 
 var _ libkbfs.Observer = (*Folder)(nil)
 
+func (f *Folder) resolve(ctx context.Context) (*libkbfs.TlfHandle, error) {
```

Comment:
I want to refactor the common bits on a higher level but want to avoid that as part of this pull request. I.e. probably have a libfs.FolderCommon and resolve etc being methods on that.